src/modules/fundingManager/bondingCurveFundingManager/ToposBondingCurveFundingManager.sol

```diff
@@ -457,12 +452,22 @@ contract ToposBondingCurveFundingManager is
     //--------------------------------------------------------------------------
     // Internal Functions
 
+    /// @dev Set the current seize state, which defines the percentage of seizable amount
+    function _setSeize(uint64 _seize) internal {
+        if (_seize > MAX_SEIZE) {
+            revert ToposBondingCurveFundingManager__InvalidSeize(_seize);
+        }
+        currentSeize = _seize;
+    }
```

Comment:
Do we want an event for updating the `currentSeize`? Just thinking in terms of our goal that we would ideally like to be able to rebuild the entire onchains state from events for potential indexing solutions.

Reply:
There is one for example for `CapitalRequiredChanged`, maybe it's more consistent to have one here as well there?

Reply:
You're right, makes sense. It wasn't in the original contracts weirdly enough.

Reply:
Quick question in regard to "being able to rebuild the on-chain state for indexing". The collateral fee collected value is internal and no event emit reflects this state. My thought process was that it shouldn't be easily retrievable for everyone how much fee has been collected. However this would be one value that is not rebuildable through emits in this way.

Reply:
Hm.. good question. I think we don't need an event for changes to `tradeFeeCollected`, seems a bit overkill for me to have one if I think about it.

As it is exposed via `getCollateralTradeFeeCollected()` in `BondingCurveFundingManagerBase `, it is not hidden though, not sure whether you meant that? In the end, an indexer can still track it as it's publically exposed via a this getter function, so we can make use of it in a potential dashboard to track fees etc. (which we probably want at some point). Users are also able to see the value easily - which to me is not a problem, I wouldn't hide it (just not rub it in their face via an event 😁)

Would you agree?

Reply:
So the `getCollateralTradeFeeCollected()` function has the `onlyOrchestratorOwner` modifier, which would inhibit it from being publicly available.

However reading your comment and seeing that you don't see an issue with users seeing the value, would it then make sense to just remove the getter function and make the `tradeFeeCollected` public, maybe also renaming it so it reflects that collateral fee is collected here?

Reply:
Oh, I missed that modifier indeed, good point! I think making the variable itself public is fine, also agree on the renaming. 👍